docs/mod-lists/general-mod-list.md

```diff
@@ -42,10 +42,28 @@ Select your Minecraft version below to view compatible mods.
     
     - Download: https://sk1er.club/mods/text_overflow_scroll
     - Support: https://discord.gg/sk1er
+
+    ### Chatting
+    Adds extremely customizable utilities such as chat tabs, screenshotting messages, and message copying
     
-    ### Sk1er Old Animations
-    Adds Minecraft 1.7 animations to 1.8.9
+    - Download: https://modrinth.com/mod/chatting
+    - Support: https://polyfrost.org/discord
+    
+    ### OverflowAnimationsV2
+    Adds 1.7 and modern Minecraft animations to 1.8.9
+    
+    - Download: https://modrinth.com/mod/animations
+    - Support: https://polyfrost.org/discord
+
+    ### OverflowAnimationsV2
+    An updated and mostly rewritten version of Sk1er's Old Animations mod. Has complete 1:1 parity with 1.7 animations, fixes most bugs and compatibility issues (including block-hit banning users from MineMenClub), and includes the custom animations feature from Dulkir and more modern version animations.
     
+    - Download: https://modrinth.com/mod/animations
+    - Support: https://polyfrost.org/discord
+
+    ### Sk1er Old Animations
+    The "original" version of OverflowAnimations, purely a 1.7 animations mod.
```

Comment:
"Original" version kinda crazy ngl

Reply:
Yeah please remove that part

Reply:
I think I was high while writing that

I retroactively apologize for that comment, will fix